Jenkinsfile

```diff
@@ -48,6 +48,10 @@ def runE2ETests(String type,String version){
         sleep 10
         node config-optic/setupqa.js
         sleep 30
+        cd test-complete-app
+        ./gradlew -i mlDeploy
+        cd ..
+        sleep 10
```

Comment:
Why the sleep here? Any change made by mlDeploy is immediately committed by MarkLogic.